routes/login.go

```diff
@@ -341,7 +341,7 @@ func (h *LoginHandler) PostResetPassword(w http.ResponseWriter, r *http.Request)
 				if err := h.mailSrvc.SendPasswordReset(user, link); err != nil {
 					conf.Log().Request(r).Error("failed to send password reset mail to %s - %v", user.ID, err)
```

Comment:
I'm not 100 % happy with the fact that part of our logging is now structured, while the other half remains plain text. I think we should definitely refactor the Sentry logging to be structured (via [samber/slog-sentry](https://github.com/samber/slog-sentry)) as well.

We can totally do that in a separate pass to not bloat further bloat this PR, though. Let me know if you'd be up for that refactoring as well (don't feel obligated, of course).

Also [samber/slog-http](https://github.com/samber/slog-http) seems like a super nice replacement for our custom request logging middleware ...

Reply:
Yes, I will need sometime to understand that lib and integrate with it. In the mean time we can have a few changes to make the sentry log structured. I will use the JSON handler instead of TextHandler for sentry, it will printed out the structured log as well
<img width="927" alt="image" src="https://github.com/user-attachments/assets/c6f3d304-32b5-447d-8cb6-99e3c9d1878f">

The other task is refactor conf.Log().Request(r).(Error | Warn | Info...) to structured. I missed this part, thanks for pointing out, probably a find and replace mistake

Reply:
I updated the Sentry slog handler to use NewJSONHandler. I tried to make it consistent with main.go to init the TextHandler for development and JSONHandler for production but no luck, I can't read the config.isDev() in this contructor. So we use JSONHandler for both in this case.

Reply:
> I can't read the config.isDev()

You could do `Get().IsDev()`, even though it's not very beautiful. Preferable option would be to pass a config from the outside.

---

What I don't like about this current implementation is the fact that a new handler gets instantiated for every single logging statement, which seems inefficient (same for the `capturingWriter` btw., which isn't optimal either).

Perhaps we could instantiate a singleton `SentryLoggerWrapper` once initially and then just reuse it inside `Log()`. Three options here:

1. Instantiate it in a `init()` method. You might not be able to call `Get()` there, though.
2. Instantiate it inside `Log()`, but only once, if not yet exiting.
3. Add some custom `InitSentryLogging()` method or so, that is called once initially (e.g. from `main()` or somewhere inside `config.go`). 

Sorry for being so nitpicky here. But I think after a technical refactoring like this, code should be at least as clean as before or cleaner.

Reply:
To minimize the changes. I choose the option `Instantiate it inside Log(), but only once, if not yet exiting.`